### PR TITLE
Add basic Gallery module using Flickity

### PR DESCRIPTION
**Pull request by josephco**
This is tied to https://github.com/CasperSleep/nightshade/pull/60, which should be merged after this

### CODE REVIEW

File: src/gallery/_gallery.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  transition: opacity 0.2s ease;
+  margin: 6px;
+  display: inline-block;
+  background-color: rgba(0, 0, 0, 0.4);
+  background-size: contain;
+}
+
+.gallery-slide-wrapper {
+  @include square(100%);
+}
```
Comment (misscs) [inline src/gallery/_gallery.scss:97]: Are we still using this file?

File: src/gallery/_base.scss [hunk 2]
```diff
@@ -1,97 +1,63 @@
+@import "https://cdnjs.cloudflare.com/ajax/libs/flickity/1.2.1/flickity.min.css";
+
```
Comment (misscs) [inline src/gallery/_base.scss]: The problem with this approach is Sass treats it as a CSS import so we take a hit with an extra HTTP request. For now, let's move a copy of the css file into our `nightshade-core` pattern folder and we can import it as we do any local css/sass file.

**Comment by misscs** [comment]
:+1: One small change otherwise DTM. 

I added another card in our backlog https://trello.com/c/PkQYzMhp. Do you agree with the problem and intent or do you not think it is that big of a pain point?

**Comment by misscs** [comment]
When you're done making the change, you can give it DTM label.

File: src/gallery/_gallery.scss (hunk 1 above)
Comment (josephco) [inline src/gallery/_gallery.scss:97]: @misscs I think it's being used on the storefront static pages, but I'm gonna strip all this out later when I use the new gallery module on those (once it's working with images).

File: src/gallery/_base.scss (hunk 2 above)
Comment (josephco) [inline src/gallery/_base.scss]: @misscs ah, good catch. I was thinking about pulling in the styles we want and adding it to the stylesheet, but for now I'll pull the file in locally and include. I think eventually we can restyle the Flickity controls ourselves, giving design an opportunity to think about them and just having our own CSS for it.

**Comment by josephco** [comment]
@misscs I agree that it may be confusing if `nightshade-core` modules rely on dependencies installed elsewhere, since we want modules to be self sufficient. Our core patterns should probably only rely on what's provided by Ando. It's something we will have to think about when we get to that card.

**Pull request merged**